pkg/session_recording/recordings.jsx

```diff
@@ -497,6 +530,13 @@
             });
         }
 
+        handleUsernameChange(username) {
+            this.setState({username: username}, () => {
+                this.clearRecordings();
+                this.journalctlRestart();
+            });
+        }
+
```

Comment:
Eh, "Submit" is OK for the POC, if we see problems, but it would be best to avoid it for production, if we can.